### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by stutisk**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,14 @@
-import "./App.css";
-import logo from "./logo.png";
+import {Home } from "./pages/Home";
+import {Routes, Route} from 'react-router-dom'
 
 function App() {
```
Comment (omjadhav85) [inline src/App.js:4]: Could change to ES6 format

**Review by junaidshaikh-js — APPROVED** [review]
Clean code, good folder structure. 

Next time I would recommend first making an initial commit with the required deletion of the files from the new project  and then push the actual code.

File: src/base.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:root{
+    --red-colorl1:#ff6584;
+    --red-colord2:#f74165;
+    /* border-radius */
+    --border-radius1:4px;
+}
+
+.padding1{
+    padding:10px 20px 10px 20px;
+}
```
Comment (omjadhav85) [inline src/base.css:10]: better to make a util.css file to have such helper classes

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <main >
+        <div className="hero">
+            <img  src={Hero} className={styles.hero} alt="hero" />
+          </div>
+            <div className="head m-a">
+                Start Your Journey !
+            </div>
+            <div className=" sub-head head m-a">
+                Every big step start with small step.
+                <br></br>
```
Comment (omjadhav85) [inline src/pages/Home.jsx:21]: use of br tag is not encouraged

**Review by omjadhav85 — APPROVED** [review]
Clean code and UI. 
Will suggest to use ES6 syntax while defining components. Minor suggestions mentioned. Also better to deploy a base version on netifly first so that you can have a deploy preview for reviewers. 
Great work!

**Comment by stutisk** [comment]
made the suggested changes , going forward with merging it

**Pull request merged**